src/CommunityToolkit.Maui.Core/Platform/Windows/MCTPopup.windows.cs

```diff
@@ -0,0 +1,255 @@
+﻿using CommunityToolkit.Core.Extensions.Workarounds;
+using CommunityToolkit.Maui.Core;
+using Microsoft.Maui.Controls.Platform;
+using Microsoft.Maui.Platform;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using LayoutAlignment = Microsoft.Maui.Primitives.LayoutAlignment;
+using XamlStyle = Microsoft.UI.Xaml.Style;
+using WindowsThickness = Microsoft.UI.Xaml.Thickness;
+using System.Diagnostics;
+
+namespace CommunityToolkit.Core.Platform;
+
+/// <summary>
+/// The navite implementation of Popup control.
+/// </summary>
+public class MCTPopup : Flyout
```

Comment:
Let's avoid using `MCT` in `CommunityToolkit.Maui` and `CommunityToolkit.Maui.Core`. 

Long story short, marketing has strict rules around branding, and "MAUI Community Toolkit" always needs to be referred to as ".NET MAUI Community Toolkit". I agree it's silly, but renaming this is easier than causing a ruckus internally. 

This also now introduces two `Popup*` classes in `CommunityToolkit.Maui.Core`. 

Here's my recommendation:
- Rename this to `Popup`
- Rename `CommunityToolkit.Maui.Core.Views.Popup` to `CommunityToolkit.Maui.Core.View.Alert` since it's exclusively used for `IAlert`